packages/flutter_timeline_view/lib/src/screens/timeline_post_creation_screen.dart

```diff
@@ -101,6 +103,15 @@ class _TimelinePostCreationScreenState
         reactionEnabled: allowComments,
         image: image,
       );
+      await Navigator.push(
```

Comment:
the timelinePostCreation needs a nullable callback with the post.
If that callback is provided we execute that instead of var newPost = await widget.service.postService.createPost(post);

In that callback we would route to the other screen. We don't want to use navigator inside of the timeline_view (only in timeline userstory)